code-collab-app/src/editorpage.js

```diff
@@ -100,6 +102,22 @@ class Editorpage extends React.Component {
     this.addEditorCell();
   }
 
+  // Event handler for download document button.
+  // e: button click event
+  handleDownloadDocumentClick = (e) => {
+    e.preventDefault();
+    const cellDivider = '\n--------------------------------------------------------------\n';
+    let documentString = '';
+    for (let i = 0; i < this.state.cellText.length; i++) {
```

Comment:
Can you please change this to `this.props.peer_data.doc_data.length`? Because I have removed the states as per @BKDuncan's recommendation.

Reply:
Uhhh, you can add a bottom border in css if you wanted a line to divide each cell. That way you can modify the colours and thickness as well 😂

Reply:
To be clear the divider is for the text file representation of the document the user can download, wasn't really thinking of doing it in html